designs/skully/src/nose.mjs

```diff
@@ -0,0 +1,203 @@
+import { pluginBundle } from '@freesewing/plugin-bundle'
+import { cheek } from './cheek.mjs'
+import { eye } from './eye.mjs'
+import { forehead } from './forehead.mjs'
+
+function draftNose({
+  Point,
+  Path,
+  points,
+  paths,
+  Snippet,
+  snippets,
+  complete,
+  sa,
+  log,
+  store,
+  paperless,
+  macro,
+  part,
+}) {
+  console.log('nose')
+  const textAttribute = 'text-xs center'
+
+  const noseSide = store.get('noseSide')
+  const noseHeight = store.get('noseHeight')
+
+  points.point0 = new Point(0, 0)
+  points.point2 = points.point0.shift(90, noseHeight)
+  points.point0Cp1 = points.point0.shift(315, noseHeight)
+  points.point2Cp2 = points.point2.shift(325, noseHeight / 3)
+
+  paths.p1 = new Path()
+    .move(points.point0)
+    .curve(points.point0Cp1, points.point2Cp2, points.point2)
+    .hide()
+
+  points.pRotate = points.point0.shift(30, noseHeight)
+
+  paths.p0p1 = new Path().move(points.point0).line(points.point2).setClass('dashed mark')
+
+  points.point1 = points.pRotate.shiftTowards(points.point2, noseHeight * -1)
+
+  points.point1Cp2 = points.point1.shift(points.point1.angle(points.point2) + 55, noseHeight / 3)
+
+  var iterations = 0
+  var pl
+  do {
+    iterations++
+
+    points.point1 = points.point1.rotate(-0.5, points.point2)
+    points.point1Cp2 = points.point1Cp2.rotate(-0.5, points.point2)
+    points.point0Cp1 = points.point0Cp1.rotate(-0.5, points.point2)
+    paths.p1 = new Path()
+      .move(points.point0)
+      .curve(points.point0Cp1, points.point1Cp2, points.point1)
+      .setText('nose' + ' (10)', textAttribute)
+
+    pl = paths.p1.length()
+  } while (iterations < 100 && pl - noseSide > 1)
+  if (iterations >= 100) {
+    log.error('Something is not quite right here!')
+  }
+
+  points.pMiddle1 = points.point2.shiftFractionTowards(points.point1, 0.5)
+
+  points.point3 = points.point1.flipX()
+  points.point3Cp1 = points.point1Cp2.flipX()
+  points.point0Cp2 = points.point0Cp1.flipX()
+  points.pMiddle2 = points.pMiddle1.flipX()
+
+  paths.p2 = new Path()
+    .move(points.point3)
+    .curve(points.point3Cp1, points.point0Cp2, points.point0)
+    .setText('nose' + ' (10)', textAttribute)
+
+  paths.seam = new Path()
+    .move(points.point0)
+    .join(paths.p1)
+    .line(points.point2)
+    .line(points.point3)
+    .join(paths.p2)
+    .close()
+
+  // Complete?
+  if (complete) {
+    points.title = points.point0.shiftFractionTowards(points.point3, 0.5)
+    macro('title', {
+      nr: 12,
+      at: points.title,
+      scale: 0.15,
+      rotation: 325,
+      title: 'nose',
+    })
+
+    snippets.n1 = new Snippet('bnotch', points.point0)
```

Comment:
notch?

Reply:
Nope, definitely `bnotch`. It needs to match up with the same `bnotch` on the cheek part.